structured_experiments/2022.03.11.testjr/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM scribble-ocr-train as tesseract_base_image
```

Comment:
looks like these testjr files aren't supposed to be here

Reply:
Sorry, it's just a template, I didn't change anything in it.  I remember this now.  I wanted to create a test experiment so that I didn't copy over the template when I pushed the experiment.

Reply:
The testjr file has been removed.

Reply:
it still looks to be here. This entire folder is stilli n the history afaik

Reply:
Hi, so if you look at the feature/extsupport_amended branch, my test folder does not exist.  I've put the screen shot below as well as the dead link to the folder, that leads nowhere.  Please let me know if I'm mistaken.  Thanks.

[structured_experiments/2022.03.11.testjr/Dockerfile](https://github.com/BloomTech-Labs/scribble-stadium-ds/pull/120/files/c53f3958f12ecb57d9f9ad8142093a3568f18767#diff-fa69ae8a2b0aca6aa79277a0bc4679fec9c4823f43ec4f5e9b3d2ec7473cf216)


![image](https://user-images.githubusercontent.com/6295098/163738943-8be31c95-bb56-4635-89ea-58b36a304796.png)